### PR TITLE
feat: add metrics for member info

### PR DESCRIPTION
**Pull request by walbertus**
Adding new metrics to get information about the members in the cluster. Current info: role and version
```
consul_serf_lan_member_info
consul_serf_wan_member_info
```

this is the proposed implementation for #299

### CODE REVIEW

**Comment by walbertus** [comment]
should I also add support for 1.16 & 1.17 in the CI?

File: .circleci/config.yml [hunk 1]
```diff
@@ -53,9 +54,9 @@ workflows:
         matrix:
           parameters:
             consul_version:
-            - "1.16"
-            - "1.17"
-            - "1.18"
+            - "1.16.6"
```
Comment (SuperQ) [inline .circleci/config.yml:57]: I don't think we want to maintain which patch release is specified in CI. This is a bit more toil than we want to maintain.

Is there a good reason to do it this way?

What about a CI step that gets the version of consul from the binary?

It looks like the Consul dockerfile includes an environment variable `PRODUCT_VERSION` which could be used.

We could also do something like this:
```
echo "CONSUL_VERSION=$(/bin/consul version | awk '$1 == "Consul" {print $2}' | tr -d 'v') >> $BASH_ENV
```
This way we can keep the minor version and detect the patch version on-demand.

See: https://circleci.com/docs/set-environment-variable/#set-an-environment-variable-in-a-shell-command

Comment (walbertus) [reply .circleci/config.yml:57]: cmiiw, this mean we don't need to configure the CONSUL_VERSION env in the circle CI config and just use PRODUCT_VERSION directly on the test?

Comment (walbertus) [reply .circleci/config.yml]: updated using $BASH_ENV, feel free to check @SuperQ

Comment (SuperQ) [reply .circleci/config.yml]: I think keeping the `CONSUL_VERSION` var makes it a little more clear to people testing outside Docker.

**Review by SuperQ — APPROVED** [review]

**Comment by walbertus** [comment]
@SuperQ any plan to trigger the release that includes this MR? @SuperQ

**Pull request merged**